rest-api/api/base_api.py

```diff
@@ -206,9 +205,25 @@ def put(self, id_, participant_id=None):
     self._do_update(m)
     return self._make_response(m)
 
+  def patch(self, id_):
```

Comment:
I'd either move this to biobank_order_dao, or define an update_with_patch method here that at a minimum throws an exception saying "implement me"... the call to update_with_patch below with no implementation here is weird.